packages/craco/README.md

```diff
@@ -237,6 +214,45 @@ module.exports = {
 };
 ```
 
+### The Mode Property  
```

Comment:
I believe it would help to read the docs about the mode property before reading the actual config sample?

Reply:
Agreed